src/api/client.ts

```diff
@@ -0,0 +1,197 @@
+/**
+ * RequestMethod
+ * The HTTP request methods that our http client supports.
+ */
+export type RequestMethod =
```

Comment:
In implementing, I found some of the line lengths on these things to get in the way. I'm curious how we @DavidLarsKetch @steven-klein feel about change `Request` to `Req` and `Response` to `Resp` for brevity sake.

Reply:
@DavidLarsKetch seems like a pretty solid idea to me.  Give a thumbs up if you're onboard.  I'm happy to implement in a separate PR, so we can see it.